src/components/BankTransactionMobileList/BusinessForm.tsx

```diff
@@ -182,7 +182,7 @@ export const BusinessForm = ({
       <div className='Layer__bank-transaction-mobile-list-item__actions'>
         {showReceiptUploads && (
           <FileInput
-            onUpload={receiptsRef.current?.uploadReceipt}
+            onUpload={(files: File[]) => receiptsRef.current?.uploadReceipt(files[0])}
```

Comment:
We use `FileInput` component in Bank Transactions. Should we support multiple file upload here too? Probably it's not necessary, but Typescript will complain about types, so we could use:

```tsx
onUpload: (files: File | File[]) => void
```